html/index.js

```diff
@@ -1,6 +1,6 @@
 /* global $ */
 /* global initialjson, feedback, AnsiUp, dayjs, FileReader, MouseEvent, HTMLAnchorElement */
-const expectedClaimVersion = 'v0.3.0'
+const expectedClaimVersion = 'v0.4.0'
```

Comment:
run a make build to also update the result.html

Reply:
Done.